bin/add_geneticmodels_to_svvcf.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+from pysam import VariantFile
+import cmdvcf
+import argparse
+from geneticmodels import GM
+
+def main(args: object):
+    """
+    Read VCF line by line, add genetic models to INFO-field
+    """
+    ped, individuals = read_ped(args.pedigree_file)
+    if len(ped.keys()) < 2:
+        exit("Cannot add genetic models to single samples")
+    vcf_object = VariantFile(args.input_vcf)
+    vcf_object.header.info.add(
+        "GeneticModel",
+        ".",
+        "String",
+        "Genetic model for variant"
+    )
+    vcf_object.header.info.add(
+        "HOMHEM",
+        ".",
+        "String",
+        "If variant follows geneticmodel and is a homo- or hemi-zygous deletion"
+    )
+    vcf_out = VariantFile(args.out_vcf, "w", header=vcf_object.header)
```

Comment:
To make the `main` function neater, you could put this into an `output_vcf` function